### PR TITLE
Updated travis and test configurations

### PR DESCRIPTION
**Pull request by othercorey**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,29 +18,22 @@ matrix:
...
+  - if [[ $PREFER_LOWEST == 1 ]]; then composer update --prefer-lowest --prefer-stable --no-interaction; fi
 
 script:
   - if [[ $DEFAULT == 1 ]]; then composer test; fi
-
-  - if [[ $PREFER_LOWEST == 1 ]]; then composer lowest; fi
-
   - if [[ $CHECKS == 1 ]]; then composer cs-check; fi
 
-cache:
```
Comment (dereuromark) [inline .travis.yml]: I would have kept this below, as this is just noise at the top

File: composer.json [hunk 2]
```diff
@@ -31,22 +31,13 @@
...
           "@add-standard",
-          "@increase-severity",
-          "phpunit",
-          "@reset-ruleset"
+          "phpunit"
         ],
-        "cs-check": "phpcs --colors -p -s --exclude=SlevomatCodingStandard.TypeHints.DeclareStrictTypes --standard=CakePHP CakePHP/Sniffs/",
-        "cs-fix": "phpcbf --colors -p --exclude=SlevomatCodingStandard.TypeHints.DeclareStrictTypes --standard=CakePHP CakePHP/Sniffs/",
-        "lowest": " validate-prefer-lowest",
-        "lowest-setup": "composer update --prefer-lowest --prefer-stable --prefer-dist --no-interaction && cp composer.json composer.backup && composer require --dev dereuromark/composer-prefer-lowest && mv composer.backup composer.json",
```
Comment (dereuromark) [inline composer.json]: this should be kept for bc check!

File: .travis.yml (hunk 1 above)
Comment (othercorey) [reply .travis.yml]: I just aligned it with cakephp.

File: composer.json (hunk 2 above)
Comment (othercorey) [reply composer.json]: Sure, if you really want it.

File: .travis.yml (hunk 1 above)
Comment (dereuromark) [reply .travis.yml]: in cake i would have done the same rather, moving irrelevant things to the bottom

Comment (othercorey) [reply .travis.yml]: as you like to say "feel free to change it if you want" :)

File: composer.json (hunk 2 above)
Comment (dereuromark) [reply composer.json]: it is important to know when we rely on the wrong sniffer repo version, which is not compatible anymore with what we use.

File: .travis.yml [hunk 3]
```diff
@@ -12,29 +14,26 @@ matrix:
...
-  - if [[ $PREFER_LOWEST != 1 ]]; then composer install --prefer-source --no-interaction; fi
-  - if [[ $PREFER_LOWEST == 1 ]]; then composer lowest-setup; fi
+install:
+  - if [[ $PREFER_LOWEST != 1 ]]; then composer install --prefer-dist --no-interaction; fi
+  - if [[ $PREFER_LOWEST == 1 ]]; then composer update --prefer-lowest --prefer-stable --no-interaction; fi
 
 script:
   - if [[ $DEFAULT == 1 ]]; then composer test; fi
-
-  - if [[ $PREFER_LOWEST == 1 ]]; then composer lowest; fi
```
Comment (dereuromark) [inline .travis.yml]: shouldnt this also be restored?

Comment (othercorey) [reply .travis.yml]: Yup, missed it.

**Pull request merged**